src/util/fhir.js

```diff
@@ -33,12 +33,12 @@ function getReference(ehrUrl, reference) {
 
 function login() {
 
-    const tokenUrl = this.state.config.auth + "/realms/" + this.state.config.realm + "/protocol/openid-connect/token"
+    const tokenUrl = (process.env.REACT_APP_AUTH ? process.env.REACT_APP_AUTH : this.state.config.auth) + "/realms/" + (process.env.REACT_APP_REALM ? process.env.REACT_APP_REALM : this.state.config.realm) + "/protocol/openid-connect/token"
```

Comment:
as far as I can tell this login function isn't used anywhere and can be deleted.